battleship.py

```diff
@@ -2,6 +2,44 @@
 
 import random
 
+import pygame
+
```

Comment:
Try not to have too many spaces between lines unless they are added by the formatter.